docs/platform-feature-parity.md

```diff
@@ -0,0 +1,37 @@
+# Platform feature-parity
+
+macOS and Windows binaries are new and considered alpha-quality. Some functionality may be missing, some may be broken. (Enhancements and bug-reports welcome, please see [#551: Multi-OS support](https://github.com/aelsabbahy/goss/issues/551))
+
+This matrix attempts to track parity across platforms.
+
+Legend:
+
+* `√` - implemented, passing in CI
+* an issue number - refer to that issue for details
+
+## Matrix
+
+| Feature | `linux` implementation + unit-tests | `linux` integration-tests | `macOS` implementation + unit-tests | `macOS` integration-tests | `Windows` implementation + unit-tests | `Windows` integration-tests |
+|:---------------|:--------|:-----------------:|:----------------:|:-----------------:|:----------------:|:-------------------:|:------------------:|
+| Assertion: `addr` | √ | √ | | | | |
```

Comment:
I can format this via a tool, but honestly I think that will just make it more fiddly to edit; tables in markdown are fiddly, so I've done the bare minimum.

I don't think this is complete, but it's illustrative. What do you think?

Reply:
This doesn't render correctly for me. Maybe a rhpo somewhere?

Reply:
Hmm.. does Linux really have unit tests?

Maybe the columns need to be:

Feature, linux, windows, osx

And symbols for fully support, partial support, and no support.

Maybe for partial support, explain the subset of attributes that don't work.

Thoughts?

Reply:
Columns sounds good to me. It works conceptually, but I think will be hard to deal with inside the table formatting. I'll make headings under the table and page links from the cells where more detail is needed. What do you think?

Reply:
How's this? It's now basically a full test-plan, ready to be filled out. I don't intend to fill that out inside this PR more than what I've done to illustrate how to do it, though; sound ok?